wrappers/services.go

```diff
@@ -901,33 +902,48 @@ func EnsureSnapServices(snaps map[*snap.Info]*SnapServiceOptions, opts *EnsureSn
 	return context.reloadModified()
 }
 
-// StopServicesFlags carries additional parameters for StopServices.
-// XXX: Rename to StopServicesOptions
-type StopServicesFlags struct {
-	Disable bool
-	ScopeOptions
-}
+// categorizeServices returns a list of system and user services
+// ordered by the same order that 'app' is passed into.
+func categorizeServices(apps []*snap.AppInfo, includeActivators bool) ([]string, []string) {
```

Comment:
I have a cleanup planned for StartServices that will use this function too, which is why the `includeActivators` part exists.

Reply:
See https://github.com/snapcore/snapd/pull/13958 for details